pipeline/plot/unit_characteristic_plot.py

```diff
@@ -155,148 +168,202 @@ def plot_unit_bilateral_photostim_effect(probe_insert_key, axs=None):
-    #     (psth.UnitPsth * ipsi_sel_units & contra_trials).fetch(order_by='unit_posy desc'),
-    #     ax=axs[1], vlines=period_starts)
+    psth_is_it = (((psth.UnitPsth & conds_i)
+                   * ephys.Unit.proj('unit_posx', 'unit_posy'))
+                  & good_unit.proj() & sel_i.proj()).fetch(
+                      order_by='unit_posy desc')
 
-    psth_i_c = [psth.UnitPsth.compute_unit_trial_psth(u, contra_trials)
-                for u in ipsi_sel_units]
-    psth_i_i = [psth.UnitPsth.compute_unit_trial_psth(u, ipsi_trials)
-                for u in ipsi_sel_units]
+    psth_is_ct = (((psth.UnitPsth & conds_c)
+                   * ephys.Unit.proj('unit_posx', 'unit_posy'))
+                  & good_unit.proj() & sel_i.proj()).fetch(
+                      order_by='unit_posy desc')
 
-    _plot_stacked_psth_diff(psth_i_i, psth_i_c, ax=axs[0], vlines=period_starts)
-    axs[1].set_title('Ipsi-selective Units')
+    psth_cs_ct = (((psth.UnitPsth & conds_c)
+                   * ephys.Unit.proj('unit_posx', 'unit_posy'))
+                  & good_unit.proj() & sel_c.proj()).fetch(
+                      order_by='unit_posy desc')
 
+    psth_cs_it = (((psth.UnitPsth & conds_i)
+                   * ephys.Unit.proj('unit_posx', 'unit_posy'))
+                  & good_unit.proj() & sel_c.proj()).fetch(
+                      order_by='unit_posy desc')
 
-def plot_ave_contra_ipsi_psth(probe_insert_key, axs=None):
-    unit_hemi = (ephys.ProbeInsertion.InsertionLocation * experiment.BrainLocation
-                 & probe_insert_key).fetch1('hemisphere')
+    _plot_stacked_psth_diff(psth_cs_ct, psth_cs_it, ax=axs[0],
+                            vlines=period_starts)
 
-    period_starts = (experiment.Period & 'period in ("sample", "delay", "response")').fetch('period_start')
+    axs[0].set_title('Contra-selective Units')
+    axs[0].set_ylabel('Unit (by depth)')
+    axs[0].set_xlabel('Time to go (s)')
 
-    # query trials
-    trial_restrictor = {'task': 'audio delay', 'task_protocol': 1, 'early_lick': 'no early', 'outcome': 'hit'}
-    contra_trials = (experiment.BehaviorTrial - experiment.PhotostimTrial & trial_restrictor
-                     & {'trial_instruction': 'right' if unit_hemi == 'left' else 'left'})
-    ipsi_trials = (experiment.BehaviorTrial - experiment.PhotostimTrial & trial_restrictor
-                   & {'trial_instruction': 'left' if unit_hemi == 'left' else 'right'})
+    _plot_stacked_psth_diff(psth_is_it, psth_is_ct, ax=axs[1],
+                            vlines=period_starts, flip=True)
 
-    ipsi_sel_units = ephys.Unit * psth.UnitSelectivity & 'unit_selectivity = "ipsi-selective"'
-    contra_sel_units = ephys.Unit * psth.UnitSelectivity & 'unit_selectivity = "contra-selective"'
+    axs[1].set_title('Ipsi-selective Units')
+    axs[1].set_ylabel('Unit (by depth)')
+    axs[1].set_xlabel('Time to go (s)')
+
+
+def plot_ave_contra_ipsi_psth(probe_insert_key, axs=None):
```

Comment:
I know I'm being picky but I fine `ave` as an shorthand for `average` to be a bit awkward - would prefer `avg`

Reply:
ah, agreed